core/src/main/java/org/apache/iceberg/util/PropertyUtil.java

```diff
@@ -18,13 +18,23 @@
  */
 package org.apache.iceberg.util;
 
+import java.util.List;
 import java.util.Map;
+import java.util.Set;
 import java.util.stream.Collectors;
+import org.apache.iceberg.TableProperties;
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableMap;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableSet;
+import org.apache.iceberg.relocated.com.google.common.collect.Maps;
 
 public class PropertyUtil {
 
+  private static final Set<String> COLUMN_PREFIX_PROPERTIES =
```

Comment:
Nit: I actually thought initially, about SchemaUpdate passing this explicitly (otherwise, there doesn't seem much use for making it a variable if its just hard-coded)

Reply:
Updated.